Sources/SwifterSwift/Dispatch/DispatchQueueExtensions.swift

```diff
@@ -41,6 +41,19 @@ public extension DispatchQueue {
 
         return DispatchQueue.getSpecific(key: key) != nil
     }
+    
+    /// SwifterSwift: Runs passed closure asynchronous after certain time interval
+    ///
+    /// - Parameter delay: The time inverval after which the closure will run.
+    /// - Parameter qos: Quality of service at which the work item should be executed.
+    /// - Parameter flags: Flags that control the execution environment of the work item.
+    /// - Parameter work: The closure to run after certain time interval.
+    func asyncAfter(delay: Double,
```

Comment:
`TimeInterval`?

Reply:
TimeInterval is just a type alias for double and is provided in Foundation module. Should I import the module for that?

Reply:
> TimeInterval is just a type alias for double

Yeah, but I'm thinking more about readability here since `TimeInternal` expresses better the concept that the method expects a time interval instead of an arbitrary double... 

> is provided in Foundation module. Should I import the module for that?

Well if Foundation is required, maybe we could leave as it is

Reply:
Totally agree with you. Just importing the module for that seems like an overkill  :)